<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchStacking</class>
 <widget class="QDialog" name="BatchStacking">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Stacking</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string comment="IDC_STATIC">This dialog is used to launch the stacking process from existing image lists.

Add the file lists you want to stack and click OK to start the stacking processes.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addLists">
       <property name="text">
        <string comment="IDC_ADDLISTS">Add File Lists...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearLists">
       <property name="text">
        <string comment="IDC_CLEARLIST">Clear List</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="fileLists">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DropOnly</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchStacking</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchStacking</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addLists</sender>
   <signal>clicked()</signal>
   <receiver>BatchStacking</receiver>
   <slot>addLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearLists</sender>
   <signal>clicked()</signal>
   <receiver>BatchStacking</receiver>
   <slot>clearLists()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLists()</slot>
  <slot>clearLists()</slot>
 </slots>
</ui>
